test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -41,22 +41,32 @@
 
 @tags('multisched')
 class TestSchedulerInterface(TestInterfaces):
+
     """
     Test suite to test different scheduler interfaces
     """
 
     def setUp(self):
         # delete all schedulers except default
-        self.server.manager(MGR_CMD_DELETE, SCHED, id="@default")
+        self.server.manager(MGR_CMD_DELETE,
+                            SCHED,
+                            id="@default")
+        self.server.manager(MGR_CMD_CREATE,
+                            SCHED,
+                            id="TestCommonSched")
         TestInterfaces.setUp(self)
 
     def test_create_successful_scheduler_object(self):
         """
         Creates a scheduler object succcessfully .
         """
-        a = {'sched_cycle_length': 12000}
-        rc = self.server.manager(MGR_CMD_CREATE, SCHED, a, id="test1")
-        self.assertTrue(rc == 0)
+        try:
+            self.server.manager(MGR_CMD_CREATE,
+                                SCHED,
+                                {'sched_cycle_length': 12000},
+                                id="testCreateSched")
+        except PbsManagerError, e:
+            self.fail(e.msg)
```

Comment:
I agree with @bhroam on both point, 1st we don't need this test case because in setUp() you calling manager with DELETE and CREATE option if that functionality is broken then setUp() will fail which means test execution flow will never reach to this test (same for test_delete_schedulers) and 2nd yes, raising exception for API was because we wanted to avoid try/except blocks from test cases, and let test fail automatically if API fails.